.pre-commit-config.yaml

```
@@ -34,7 +34,7 @@ repos:
       exclude: '^(.typos.toml|src/SmithWaterman.cc|testing/.*|auxil/.*|scripts/base/frameworks/files/magic/.*|CHANGES)$'
 
 - repo: https://github.com/bbannier/spicy-format
-  rev: v0.15.0
+  rev: v0.16.1
```

Please bump to 0.16.2, 0.16.1 had another dependency issue.
```suggestion
  rev: v0.16.2
```